lib/venue.rb

```diff
@@ -1,2 +1,38 @@
 class Venue
+
+  attr_reader :name, :capacity, :patrons #:yell_at_patrons
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @patrons = []
+    #@yell_at_patrons =[]
+  end
+
+  def add_patron(patron)
+    @patrons << patron
+  end
+
+  def yell_at_patrons
+    @patrons.map(&:upcase) #found on google when I couldn't get.each to work
+   # @yell_at_patrons = []
+   # patrons.each do |patron|
```

Comment:
This was a great start using the `.each` you were so close! Remember that Ruby runs top to bottom. So all you would need to do to get this to work is call `yell_at_patrons` as the last line. So it might look like: 
```
def yell_at_patrons
   yelled_at = []
   @patrons. each do |patron|
       yelled_at << patron.upcase
    end
    yelled_at
end
```